Source/ProgressionSystemRuntime/Public/Components/PSHUDComponent.h

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "Data/PSTypes.h"
+#include "Data/PSDataAsset.h"
```

Comment:
Please remove `#include "Data/PSDataAsset.h"` because of all class pointers can be 'forwarded' with 'class` before pointer type and you even don't use any PSDataAsset in this .h

Reply:
removed